platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HttpRequestUrl.java

```diff
@@ -25,10 +28,10 @@ public static String buildResourceUrl(@NonNull String host, String resourceUrl,
       } else {
         resourceUrl = resourceUrl + "&";
       }
-      resourceUrl = resourceUrl + "sku=" + Mapbox.getSkuToken();
-
       if (offline) {
-        resourceUrl = resourceUrl + "&offline=true";
+        resourceUrl = resourceUrl + "offline=true";
```

Comment:
nit: add inline documentation to this part of the code:

>             // Only add SKU token to requests not tagged as "offline" usage.

Reply:
Added in https://github.com/mapbox/mapbox-gl-native/pull/15363/commits/99a613155a609f3a023b3fd4a94a78876f9d1b59